src-vue/src/views/DeveloperView.vue

```diff
@@ -8,6 +8,10 @@
             Panic button
         </el-button>
 
+        <el-button type="primary" @click="checkLinuxCompatability">
+            Check NSProton Compatability
+        </el-button>
```

Comment:
In the future the Linux compat check should be done as part of NSProton install and not be a button but for now it's fine as it's hidden in the dev view anyway ^^